app/v1/models/request/SubmitReturnRequestBody.scala

```diff
@@ -46,25 +45,23 @@ object SubmitReturnRequestBody {
       (JsPath \ "totalValueGoodsSuppliedExVAT").read[BigDecimal] and
       (JsPath \ "totalAcquisitionsExVAT").read[BigDecimal] and
       (JsPath \ "receivedAt").read[String] and
-      (JsPath \ "uniqueId").read[String] and
-      Reads.pure(None)
+      (JsPath \ "uniqueId").read[String]
     ) (SubmitReturnRequestBody.apply _)
```

Comment:
We don't need an explicit reads since the model already matches the expected format.